packages/server/src/typeDefs.js

```diff
@@ -0,0 +1,50 @@
+import { gql } from "apollo-server";
+
+export const typeDefs = gql`
+  type Pastry {
+    id: ID!
+    name: String!
+    origin: String!
+    imgsrc: String
+    description: String!
+    price: Float!
+    categoryID: ID
+    category: Category
+  }
+
+  type Category {
+    id: ID!
+    name: String!
+    pastries: [Pastry]
+  }
+
+  type Query {
+    pastries: [Pastry]
+    pastry(id: ID!): Pastry
+    categories: [Category]
+  }
+
+  input PastryInput {
+    name: String!
+    origin: String!
+    imsrc: String
```

Comment:
This should have been imgsrc and it was tripping up issues with submitting the mutation.